### PR TITLE
Call het deletions in GermlineThresholdCaller

### LINKED ISSUE #302: Multiple reference bases found in sample
When I try to run `germline-threshold` on `brody_307-bcell-tumor-bwa-mem-gap11-gep4-dedup-indelrealigned-bqsr.bam`, I eventually get the following error:

`java.lang.IllegalArgumentException: Multiple reference bases found in sample = SM at (chr, pos) = (17, 4455004)`

Full command line:
```
spark-submit --master yarn --deploy-mode cluster --driver-java-options -Dlog4j.configuration=scripts/log4j.properties --executor-memory 4g --driver-memory 10g --num-executors 1000 --executor-cores 1 --class org.hammerlab.guacamole.Guacamole --verbose target/guacamole-with-dependencies-0.0.1-SNAPSHOT.jar germline-threshold --reads hdfs:///user/ahujaa01/brody-lymphoma/brody_307/brody_307-bcell-tumor-bwa-mem-gap11-gep4-dedup-indelrealigned-bqsr.bam --out hdfs:///user/vanded03/guac-test.vcf
```

See http://demeter-mgmt1.demeter.hpc.mssm.edu:8088/cluster/app/application_1432740718700_2196

@arahuja says that this shouldn't be happening.

### PR DESCRIPTION
**Pull request by danvk**
This is my first substantive Guacamole and Scala PR, so please be brutal :) I don't know if this is the right approach.

Fixes #302

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/guacamole/304)

### CODE REVIEW

File: src/main/scala/org/hammerlab/guacamole/commands/GermlineThresholdCaller.scala [hunk 1]
```diff
@@ -115,9 +115,8 @@ object GermlineThreshold {
           }
 
           sortedAlleles match {
-            /* If no alleles are above our threshold, we emit a NoCall variant with the reference allele
-           * as the variant allele.
-           */
+            // If no alleles are above our threshold, we emit a NoCall variant with the reference allele
+            // as the variant allele.
```
Comment (arahuja) [inline src/main/scala/org/hammerlab/guacamole/commands/GermlineThresholdCaller.scala]: any reason for the comment switch?

Comment (danvk) [inline src/main/scala/org/hammerlab/guacamole/commands/GermlineThresholdCaller.scala]: It was mis-indented so I was changing it anyway. I've always used `//` for inline comments and `/* .. */` exclusively for scaladoc/javadoc comments. Do you guys have any conventions around this?

**Comment by danvk** [comment]
I changed `GermlineThresholdCaller` to skip het dels. I can now run this caller on real BAMs on demeter.

@arahuja good to merge?

File: src/main/scala/org/hammerlab/guacamole/commands/GermlineThresholdCaller.scala (hunk 1 above)
Comment (arahuja) [inline src/main/scala/org/hammerlab/guacamole/commands/GermlineThresholdCaller.scala]: We don't really, so we can use this as the convention

**Comment by arahuja** [comment]
LGTM

**Pull request merged**